metals/src/main/scala/scala/meta/internal/metals/Indexer.scala

```diff
@@ -121,8 +121,10 @@ final case class Indexer(
 
     bspSession() match {
       case None =>
-        scribe.warn("No build session currently active to reload.")
-        Future.successful(BuildChange.None)
+        scribe.warn(
+          "No build session currently active to reload. Attempting to reconnect."
+        )
+        reconnectToBuildServer()
```

Comment:
If this now a part of a happy path, I don't think we should emit a warning here.

Reply:
It's part of the happy path only on initialize. I think its best to leave the warning here, its still unhappy path is something breaks.